client/src/Components/RecordForm.jsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from 'react';
+import Button from 'react-bootstrap/esm/Button';
+import { ItemForms } from './ItemsForms.jsx';
+
+export default function RecordForm() {
+  const [items, setItems] = useState(1);
+  const [out, setOut] = useState(true);
+
+  function handleItems (e) {
+    const numberOfItems = e.target.value;
+    setItems(numberOfItems);
+  }
+
+  async function handleSubmit (e) {
+
+    try {
+      e.preventDefault();
+      const form = new FormData(e.target)
+      const response = await fetch('/api/record', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(Object.fromEntries(form.entries()))
+      });
+      if(!response.ok) throw new Error(`Server error: ${response.status}`)
+      e.target.reset();
```

Comment:
Is this the UI you want? Just clear the form so they can enter a new record? If so, great, but typically you would navigate to a page that shows the results, or would close the form and display the results in the current page.